examples/clean-state/src/index.js

```diff
@@ -0,0 +1,19 @@
+import { NearContract, NearBindgen, call, view, near, LookupMap } from 'near-sdk-js'
+
+@NearBindgen
+class CleanState extends NearContract {
+    constructor(keys) {
+        super()
+        keys.forEach(key => near.jsvmStorageRemove(key))
+    }
```

Comment:
I suggest to make a separate method `cleanState`, would be more clear than reuse the constructor

Reply:
Makes sense, I just followed the Rust example which used an `#[init]` method